RedHatInsights/curiosity-frontend
# build: update changelog release script

## What's included

- build: update changelog release script

### Notes
- minor adjustment for the CHANGELOG.md release process



## How to test

Confirm the release script generates a log
1. update the NPM packages with `$ npm install`
1. `$ npm run release:rc`
   - confirm the console output is expected









## Example

...

## Updates issue/story

ongoing
